src/components/Chakra/Box.stories.mdx

```diff
@@ -0,0 +1,98 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs/blocks";
+import { Box } from "@chakra-ui/react";
+
+import { getCategory } from "../../utils/componentCategories";
+import Card from "../Card/Card";
+import Image from "../Image/Image";
+import Link from "../Link/Link";
+import { LinkTypes } from "../Link/LinkTypes";
+import Heading from "../Heading/Heading";
+import Icon from "../Icons/Icon";
+import { IconNames, IconRotationTypes } from "../Icons/IconTypes";
+
+<Meta title={getCategory("Box")} component={Box} />
+
+# Box
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `0.24.0`   |
+
+Note: This needs the use of the `DSProvider` component. See the README for more information.
+
+This component is directly exported from Chakra UI. The `Box` is a simple and
+composable component that can be used for many situations. By default it renders
+a `div` but can render another element using the `as` prop.
+
+All available props can be found on the [Style Props](https://chakra-ui.com/docs/features/style-props)
```

Comment:
This is fine for now, but as we progress with this integration, we should tie this down a bit so we can control the styling.

Reply:
We'll need to discuss this. Doing so would go against Chakra's UI patterns and expected props available, but we do need a layer between 

dev -> custom styling -> DS/Chakra components